app.js

```diff
@@ -1,3 +1,103 @@
+// FUNCTIONS
+function displayLuxonDate() {
+  // Display Date
+  /* eslint-disable */
+  const DateTime = luxon.DateTime;
+  /* eslint-enable */
+  const now = DateTime.now();
+  const dateText = now.toLocaleString(DateTime.DATETIME_MED);
+  const spanForText = document.querySelector('#luxonDate');
+  spanForText.textContent = dateText;
+}
+
+function displayList() {
+  const divList = document.querySelector('#div4list');
+  const divForm = document.querySelector('#div4form');
+  const divContact = document.querySelector('#div4contact');
+
+  // Remove d-none from divList in case it has it
+  const classesDiv = divList.className;
+  divList.className = classesDiv.replaceAll('d-none', '');
+
+  // Add d-none to divForm and divContact
+  divForm.classList.add('d-none');
+  divContact.classList.add('d-none');
+
+  // Change active link class
+  const listA = document.querySelector('#listA');
+  const formA = document.querySelector('#formA');
+  const contactA = document.querySelector('#contactA');
+
+  // Remove text-white for active class
+  let classesA = listA.className;
+  listA.className = classesA.replaceAll('text-white', 'active');
+
+  // Remove active class
+  classesA = formA.className;
+  formA.className = classesA.replaceAll('active', 'text-white');
+  classesA = contactA.className;
+  contactA.className = classesA.replaceAll('active', 'text-white');
+}
+
+function displayForm() {
+  const divList = document.querySelector('#div4list');
+  const divForm = document.querySelector('#div4form');
+  const divContact = document.querySelector('#div4contact');
+
+  // Remove d-none from divForm in case it has it
+  const classesDiv = divForm.className;
+  divForm.className = classesDiv.replaceAll('d-none', '');
+
+  // Add d-none to divList and divContact
+  divList.classList.add('d-none');
+  divContact.classList.add('d-none');
+
+  // Change active link class
+  const listA = document.querySelector('#listA');
+  const formA = document.querySelector('#formA');
+  const contactA = document.querySelector('#contactA');
+
+  // Remove text-white for active class
+  let classesA = formA.className;
+  formA.className = classesA.replaceAll('text-white', 'active');
+
+  // Remove active class
+  classesA = listA.className;
+  listA.className = classesA.replaceAll('active', 'text-white');
+  classesA = contactA.className;
+  contactA.className = classesA.replaceAll('active', 'text-white');
+}
+
+function displayContact() {
+  const divList = document.querySelector('#div4list');
+  const divForm = document.querySelector('#div4form');
+  const divContact = document.querySelector('#div4contact');
+
```

Comment:
:heavy_check_mark:  Your page navigation works as expected. :+1: 
:heavy_check_mark: The design of the page looks really neat as well. :clap: Great job. 

## Required changes:

- [ ] Your page layout is missing the copyright section provided in the wireframe template. Please, let's complete the requirement by adding the copyright section. :pray: :pray: 
KIndly see below images:

Provided wireframe
![Screenshot_2021-09-17_11-10-26](https://user-images.githubusercontent.com/70150423/133774011-918ffd68-c3fb-435a-979d-8a678ee661d2.png)

Your design
![Screenshot 2021-09-17 at 11-12-17 Awesome Books](https://user-images.githubusercontent.com/70150423/133774086-cc2d3eff-5a8b-4224-acab-95f1607a4b42.png)

[OPTIONAL]

- [ ] Let's limit the code comments so as not to risk making the codebase look untidy. Let's comment only where it is necessary.
- [ ] Eslint was disabled on many occasions throughout the code. Please, let's try as much as possible to resolve them and in so doing, learning valuable new concepts.
- [ ] Always favor the use of ES6 syntax. For example, let's make use of arrow functions instead of using the `function ` keyword.